ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1888,12 +1888,13 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot
-- as: vsphere-upi-p3-f14
-  cron: 32 3 5,20 * *
+- as: vsphere-upi-firmware-bios-p3-f14
```

Comment:
how about rename the job name to `biso` instead of `firmware-bios` for a shorter name.

Reply:
ok. let me update